clang/test/Driver/darwin-embedded-search-paths.c

```diff
@@ -0,0 +1,55 @@
+// UNSUPPORTED: system-windows
+//   Windows is unsupported because we use the Unix path separator `/` in the test.
+
+// Add default directories before running clang to check default
+// search paths.
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: cp -R %S/Inputs/MacOSX15.1.sdk %t/
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/include
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/include/c++/v1
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/local/include
+// RUN: mkdir -p %t/MacOSX15.1.sdk/embedded/usr/include
+// RUN: mkdir -p %t/MacOSX15.1.sdk/embedded/usr/local/include
```

Comment:
The "usual" pattern for this is to put `.keep` files in a folder structure in e.g. `clang/test/Driver/Inputs/darwin-embedded-search-paths/MacOSX15.1.sdk/...`, instead of creating them dynamically.

Reply:
I copied that from clang/test/Driver/darwin-subframeworks.c. Should I get rid of theses and add `.keep` files for both?

Reply:
I think that would be an improvement... tiny bit less disk traffic during test runs.

Reply:
Alright I'll do that